src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package model;
+
+public class FruitTransaction {
+    private final Operation operation;
+    private final String fruit;
+    private final int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public enum Operation {
```

Comment:
The Operation enum is tightly coupled with the FruitTransaction class. If the operations need to be used in other contexts, consider moving it to its own file to enhance reusability and maintainability.